testing/unit-tests.php

```diff
@@ -32,14 +23,14 @@
 require 'unit-tests/_files.php';
 
 // Setup test files
-$test = new GroupTest('CSSTidy unit tests');
+$test = new TestSuite('CSSTidy unit tests');
 foreach ($test_files as $test_file) {
 	require_once "unit-tests/$test_file";
 	list($x, $class_suffix) = explode('.', $test_file);
-	$test->addTestClass("csstidy_test_$class_suffix");
+	$test->add("csstidy_test_$class_suffix");
```

Comment:
And this method was changed: https://github.com/simpletest/simpletest/commit/d6fd5980824f30163bf4f45e360d1a64fd75e436#diff-4965ba7b1e2c14d822671009f5783ff8L519